src/ssmDocument/commands/openDocumentItem.ts

```diff
@@ -0,0 +1,134 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import { SSM } from 'aws-sdk'
+import * as vscode from 'vscode'
+import { DocumentItemNode } from '../explorer/documentItemNode'
+import { AwsContext } from '../../shared/awsContext'
+import { getLogger, Logger } from '../../shared/logger'
+import * as telemetry from '../../shared/telemetry/telemetry'
+import * as picker from '../../shared/ui/picker'
+
+export interface SsmDocumentQuickPickItem {
```

Comment:
What benefit does this interface provide over `vscode.QuickPickItem`?

Reply:
Will look into QuickPickItem